src/svgcontext.ts

```diff
@@ -487,25 +517,24 @@ export class SVGContext {
     this.path +=
       'M' + x1 + ' ' + y1 + ' A' + radius + ' ' + radius + ' 0 ' + largeArcFlag + ' ' + sweepFlag + ' ' + x2 + ' ' + y2;
     if (!isNaN(this.pen.x) && !isNaN(this.pen.y)) {
-      this.peth += 'M' + this.pen.x + ' ' + this.pen.y;
+      this.path += 'M' + this.pen.x + ' ' + this.pen.y; // BUG? this.peth => this.path
```

Comment:
@ronyeh did you get any difference using
`npm run test:reference`

Reply:
Yikes! You can grep for functions that use this and see if there are any tests that would be affected.

Reply:
I added console.logs around that line....

So the containing method `arcHelper(...)` is called 1560 times, but every single time this.pen.x and this.pen.y are both `NaN`.

So this buggy line of code was never used.

```
    this.pen.x = x;
    this.pen.y = y;
```

The above code only appears in methods like: moveTo, lineTo, bezierCurveTo, quadraticCurveTo.

So the buggy line of code would have only been run if arc() is called after one of the above methods. However, in the code base, arc is only called right after beginPath(), so this.pen.x and this.pen.y are both NaN.

Also, arc() is only ever used to draw circles.

I think the intention of this line was to move the SVG path's current point back to where we were before we decided to draw the dot using arc().

So if we did: moveTo, lineTo, arc, lineTo....  then the second lineTo would continue where the first lineTo left off.